frontend/components/JoinUsComponent/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Heading } from '../shared/Heading';
+import Button from '../shared/Button';
+
+const JoinUs = ({ data }) => {
+  const {
+    title,
+    description,
+    mentorBtn,
+    memberBtn,
+    left_img,
+    right_img
+  } = data;
+  return (
+    <div className="border-4 border-dashed border-c100">
+      <div className="flex justify-around">
+        <div className="my-24 w-1/6">
+          <img
+            className="object-cover w-11/12 hidden md:block"
+            src={left_img}
+            alt=""
+          />
+        </div>
+        <div className="w-4/6 flex flex-col justify-center mx-auto mb-24">
+          <Heading
+            type="cardSmall"
+            className="text-center mt-16 py-5 uppercase text-lg font-black"
+          >
+            {title}
+          </Heading>
+
+          <p className="join-us-description text-center pt-8 text-xs text-c500">
+            {description}
+          </p>
+
+          <div className="mt-8 mx-auto">
+            <Button
```

Comment:
you should wrap the button component with Link component for the page but don't do that now we will talk about that later